inductiva/tasks/methods.py

```diff
@@ -51,51 +48,6 @@ def _fetch_tasks_from_api(
             raise e
 
 
-def _dataframe_of_tasks(tasks: Sequence["inductiva.tasks.Task"]) -> str:
```

Comment:
See my comments below, but I think this should remain here, since it can be used both in Python and CLI.

Reply:
We're not using dataframes anywhere, so there's no point in keeping the method. But we need to be sure we're updating all possible references to this method and making sure the caller handles the possibles changes brought in by the change.

Reply:
Yes, I meant the method that substitutes this.

Reply:
This method was only being called in one place.